OpenSim/Common/Component.h

```diff
@@ -907,6 +957,11 @@ OpenSim_DECLARE_ABSTRACT_OBJECT(Component, Object);
     /**
     * Get the Output provided by this Component by name.
     *
+    * <b>C++ example:</b> get an Output from a Component in a model
+    * @code{.cpp}
+    * model.getComponent("/path/to/component").getOutput("outputName");
+    * @endcode
+    *
     * @param name   the name of the cache variable
```

Comment:
"Cache variable" definitely is confusing and should be changed "output." But I do agree it could be handled in a separate PR.